modules/auxiliary/fuzzers/http/http_post_strings.rb

```diff
@@ -0,0 +1,66 @@
+require 'msf/core'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Auxiliary::Fuzzer
```

Comment:
You could try to leverage the mixin like the other fuzzers do. It was built for this.